docs/content/dagster-plus/deployment/agents/kubernetes/configuring-running-kubernetes-agent.mdx

```diff
@@ -73,16 +73,23 @@ In this step, you'll spin up the agent with Helm.
    helm repo update
    ```
 
-2. Install the agent chart, specifying:
+2. Create a `values.yaml` file, indicating the Dagster+ deployment the agent will be responsible for. **Note**: When your Dagster+ organization is first created, the default deployment is `prod`:
+
+   ```yaml
+   dagsterCloud:
+     deployment: prod
+   ```
+
+3. Install the agent chart, specifying:
 
    - The namespace into which to install the chart
-   - The Dagster+ deployment the agent will be responsible for. **Note**: When your Dagster+ organization is first created, the default deployment is `prod`:
+   - The `values.yaml` file created in the previous step.
```

Comment:
```suggestion
   - The `values.yaml` file created in the previous step
```

Nit - no period needed here, since the list item before it doesn't have one. Removing for consistency.